versions.txt

```diff
@@ -29,3 +29,6 @@ autoinstrumentation-dotnet=0.5.0
 # Represents the current release of Apache HTTPD instrumentation.
 # Should match autoinstrumentation/apache-httpd/version.txt
 autoinstrumentation-apache-httpd=1.0.2
+
+# Represents the current release of the Operator OpAMP Bridge.
+operator-opamp-bridge=0.1.0
```

Comment:
Can we move this up next to the target allocator and match the version as well?

Reply:
this is why unit tests are failing as well. 
```
Error: invalid command options: 0.68.0 0.1.0 is not a valid semantic version: Invalid character(s) found in patch number "0 0.1.0"
```

Reply:
ah actually this may be an issue with the grep happening in the action.

Reply:
namely this grep in the Makefile:
```
OPERATOR_VERSION ?= "$(shell grep -v '\#' versions.txt | grep operator | awk -F= '{print $$2}')"
```

Reply:
Yes. As `operator` word is present in `operator-opamp-bridge=0.1.0` also, so it is causing the problem. 
Should we use `grep operator=` here so that would not conflict with `operator-opamp-bridge=0.1.0` ?